gsoc/urls.py

```diff
@@ -25,6 +26,7 @@
 #Add Django site authentication urls (for login, logout, password management)
 urlpatterns += [
     url('accounts/', include('django.contrib.auth.urls')),
+    url('irc_test/', views.irc_test),
```

Comment:
remove debugging before committing please 👍

Reply:
Sorry my bad! I totally forgot about it.